### ISSUE TITLE
Created report but NPM throw error

### ISSUE THREAD

**Issue by chintan9** [mportuga/eslint-detailed-reporter#20]
**Describe the bug**
Created ESLint Error report but NPM throw error.

**To Reproduce**
run this code in terminal 
`eslint . -f node_modules/eslint-detailed-reporter/lib/detailed.js -o report.html`

**Expected behavior**
Should not throw error. 
**Screenshots**

**Desktop (please complete the following information):**
log file 

```
   1   │ 0 info it worked if it ends with ok
   2   │ 1 verbose cli [
   3   │ 1 verbose cli   '/usr/local/Cellar/node@12/12.14.1_1/bin/node',
   4   │ 1 verbose cli   '/usr/local/bin/npm',
   5   │ 1 verbose cli   'run',
   6   │ 1 verbose cli   'lint:report'
   7   │ 1 verbose cli ]
   8   │ 2 info using npm@6.13.6
   9   │ 3 info using node@v12.14.1
  10   │ 4 verbose run-script [ 'prelint:report', 'lint:report', 'postlint:report' ]
  11   │ 5 info lifecycle project@1.0.3~prelint:report: project@1.0.3
  12   │ 6 info lifecycle project@1.0.3~lint:report: project@1.0.3
  13   │ 7 verbose lifecycle project@1.0.3~lint:report: unsafe-perm in lifecycle true
  14   │ 8 verbose lifecycle project@1.0.3~lint:report: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecy
       │ cle/node-gyp-bin:/Users/myname/Playground/project/node_modules/.bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin
       │ :/sbin:/usr/local/opt/node@12/bin:/usr/local/opt/qt/bin:/usr/local/sbin
  15   │ 9 verbose lifecycle project@1.0.3~lint:report: CWD: /Users/myname/Playground/project
  16   │ 10 silly lifecycle project@1.0.3~lint:report: Args: [
  17   │ 10 silly lifecycle   '-c',
  18   │ 10 silly lifecycle   'eslint . -f node_modules/eslint-detailed-reporter/lib/detailed.js -o report.html'
  19   │ 10 silly lifecycle ]
  20   │ 11 silly lifecycle project@1.0.3~lint:report: Returned: code: 1  signal: null
  21   │ 12 info lifecycle project@1.0.3~lint:report: Failed to exec lint:report script
  22   │ 13 verbose stack Error: project@1.0.3 lint:report: `eslint . -f node_modules/eslint-detailed-reporter/lib/de
       │ tailed.js -o report.html`
  23   │ 13 verbose stack Exit status 1
  24   │ 13 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle
       │ /index.js:332:16)
  25   │ 13 verbose stack     at EventEmitter.emit (events.js:223:5)
  26   │ 13 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle
       │ /lib/spawn.js:55:14)
  27   │ 13 verbose stack     at ChildProcess.emit (events.js:223:5)
  28   │ 13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
  29   │ 13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
  30   │ 14 verbose pkgid project@1.0.3
  31   │ 15 verbose cwd /Users/myname/Playground/project
  32   │ 16 verbose Darwin 18.7.0
  33   │ 17 verbose argv "/usr/local/Cellar/node@12/12.14.1_1/bin/node" "/usr/local/bin/npm" "run" "lint:report"
  34   │ 18 verbose node v12.14.1
  35   │ 19 verbose npm  v6.13.6
  36   │ 20 error code ELIFECYCLE
  37   │ 21 error errno 1
  38   │ 22 error project@1.0.3 lint:report: `eslint . -f node_modules/eslint-detailed-reporter/lib/detailed.js -o re
       │ port.html`
  39   │ 22 error Exit status 1
  40   │ 23 error Failed at the project@1.0.3 lint:report script.
  41   │ 23 error This is probably not a problem with npm. There is likely additional logging output above.
  42   │ 24 verbose exit [ 1, true ]

 ```
 **Additional context**

**Comment by vseverestek** [comment]
Having same issues.
Any fix on this?
